setup.cfg

```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
+universal=1
```

Comment:
I know this is a little pedantic, but could we add the newline at the end of this file?